### DISCUSSION TITLE
Huge WASM blob

### DISCUSSION THREAD

**Discussion by ghost** [DioxusLabs/dioxus#449] [General]
`dioxus create`, then `dioxus build --release` creates a file of size 2 megabytes in the dist directory. Is this the norm? Somewhere in the documentation it says about 60 kilobytes

**Comment by jkelleyrtp** [comment]
The CLI supports wasm-opt but it's not enabled by default, unfortunately. We will likely change this behavior in a future version.

Wasm-opt will shrink 2mb down to about 500k and then gzip will shrink that down to about 60k over the network.

To enable wasm-opt, add this line to your DIoxus.toml:

https://github.com/DioxusLabs/cli/blob/d7d4699606279798ec81304d77bb619a65a9ae78/Dioxus.toml#L45
